.changes/unreleased/BUG FIXES-20240325-101108.yaml

```
@@ -0,0 +1,6 @@
+kind: BUG FIXES
+body: 'internal/fwserver: Ensure computed dynamic values are marked as dynamic type
+  during plan modification.'
```

Nit: Should this changelog focus more on the error message its preventing for developers? e.g.

```suggestion
body: 'resource: Ensured computed-only dynamic attributes will not always cause `wrong final
  value type` errors during planning'
```

---

sounds good to me 👍🏻